src/routes/backoffice.ts

```diff
@@ -458,16 +458,32 @@ export const generateBookingForPerson = (bookingData) => {
     name: `mocksolaris-transaction-${purpose}`,
     recipient_bic: recipientBIC,
     recipient_iban: recipientIBAN,
-    recipient_name: recipientName,
+    recipient_name: getRecipientName(),
```

Comment:
if you swap the recipient with the sender then make sure to take all the props not just the name